### ISSUE TITLE
how to parse training examples with 2 dimension in csv file with tf.decode_csv

### ISSUE THREAD

**Issue by jingmeng-bioinformatics** [tensorflow/tensorflow#19172]
I used tf.decode_csv in tensorflow as decoder to parse training examples in a tab-delimited file into cnn models. For every training example, the features are 2 dimensions (100 columns, 2000 rows). After reading the document in tensorflow official site, I still have two questions.

1. how to create record_defaults for my case (2 dimensional data) and how to stack the features of 2 dimensionality? The following is my code to do that, but it gave me errors.

filename_queue = tf.train.string_input_producer([file], num_epochs)

key, value = tf.TextLineReader().read(filename_queue)

record_defaults = [[1.0] for col in range(0, 100)]

content = tf.decode_csv(value, record_defaults = record_defaults, field_delim = '\t')

features = tf.stack(content[0:100])

 2. I am doing binary (0, 1) classification. Where do I put the labels for training examples? in the 2001th row? (For every training example, the first 2000 rows for features, and the 2001th row for label)

Thanks for your time!

Have I written custom code    Yes
OS Platform and Distribution    ubuntu 16.04 LTS
TensorFlow installed from          pip3 install tensorflow-gpu
TensorFlow version       the latest version
Bazel version     NA
CUDA/cuDNN version     CUDA Toolkit 9.0  
GPU model and memory     GeForce GTX 1080
Exact command to reproduce   NA

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 16 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?
